### ISSUE TITLE
beginner/basics/optimization_tutorial.html 오탈자 번역

### ISSUE THREAD

**Issue by Taeyoung96** [9bow/PyTorch-tutorials-kr#252]
## 문서 URL
_수정이 필요한 URL을 남겨주세요. (예. https://tutorials.pytorch.kr/beginner/basics/intro.html)_
- **URL**: https://tutorials.pytorch.kr/beginner/basics/optimization_tutorial.html  

## 변경 사항
_(1)어떤 단어 / 문장 / 내용이 (2)어떻게 변경되어야 한다고 생각하세요?_  

Learning rate를 '학습율'로 번역을 진행했는데, 통일성을 위해 '학습률'로 번역을 진행해야 생각합니다.  

## 추가 정보
_위와 같이 생각하신 이유 또는 다른 참고할 정보가 있다면 알려주세요._

[번역 규칙](https://github.com/9bow/PyTorch-tutorials-kr/blob/master/TRANSLATION_GUIDE.md)에 나와있는 learning rate, lr 번역 방법을 참고했습니다.
